docs/vm/create-vm.md

```diff
@@ -153,6 +153,12 @@ A disk can be made accessible via the following types:
 
 A volume's [StorageClass](../advanced/storageclass.md) can be specified when adding a new empty volume; for other volumes (such as VM images), the `StorageClass` is defined during image creation.
 
+:::info important
+
+When creating volumes from VM images, be sure the volume size is sufficient for the image. This is particularly important with qcow2 images, which can have a virtual size larger than the physical image size. If the volume size is set smaller than size of the underlying image, the volume will potentially be corrupt. By default, Harvester will set the volume size to 10GiB, or the virtual size of the image, whichever is greater.
```

Comment:
```suggestion
When creating volumes from a VM image, ensure that the volume size is greater than or equal to the image size. This is particularly important for qcow2 images because the virtual size is typically greater than the physical size. The volume may become corrupted if the configured volume size is less than the size of the underlying image.

By default, Harvester sets the volume size to either 10 GiB or the virtual size of the VM image, whichever is greater.
```

Reply:
I wouldn't have understood the meaning of "sufficient" if I didn't read your comment so I used "greater than or equal to" for clarity.